RiotSharp.Test/RiotApiTest.cs

```diff
@@ -282,7 +282,7 @@ public void GetLeaguesAsync_BySummoner_Test()
             {
                 var leagues = api.GetLeaguesAsync(RiotApiTestBase.summonersRegion, RiotApiTestBase.summonerIds.FirstOrDefault());
 
-                Assert.IsNotNull(leagues);
+                Assert.IsTrue(leagues.Count > 0);
```

Comment:
As I said, I went back to this version. I'm sure this assert is okay when i was not with the other.

Reply:
You could use both of the asserts, so you get an assert based error message. But technically it doesn't matter.

Reply:
Interesting. 
And @JanOuborny except this, the code is okay or I need to do some more modifications ?

Reply:
I haven't checked it, yet.